doc/tutorials/making_your_first_app.rst

```diff
@@ -392,13 +392,13 @@ This can be done by sublassing :py:class:`armi.plugins.UserPlugin`:
 
     class UserPluginExample(plugins.UserPlugin):
         """
-        This plugin flex-tests the onProcessCoreLoading() hook, and
+        This plugin flex-tests the afterLoadDB() hook, and
         arbitrarily adds "1" to the power ever each fuel block.
         """
 
         @staticmethod
         @plugins.HOOKIMPL
-        def onProcessCoreLoading(core, cs):
+        def afterLoadDB(core, cs):
```

Comment:
Why change this? It seems like the docs before were still valid.

Reply:
I just wanted a working example.